example/WORKSPACE

```diff
@@ -60,28 +60,37 @@ http_archive(
 
 load("@rules_conda//:defs.bzl", "conda_create", "load_conda", "register_toolchain")
 
+USE_MAMBA = True
+CLEAN = True
+
 # download and install conda
 load_conda(
+<<<<<<< 5fbf322654ac9ba43e9d85d5fb3239990ea7b41d
```

Comment:
issue: This is noise from git rebasing. It does not belong in the example WORKSPACE file.

suggestion: Delete this line, and double check that the example still runs.